assets/js/blocks/store-notices/edit.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+const StoreNotices = () => {
+	return (
+		<div className="woocommerce-notices-wrapper">
+			<div className="woocommerce-info">
+				{ __(
+					'This is an example notice. Notices added by WooCommerce or extensions will show up here.',
+					'woo-gutenberg-products-block'
+				) }
+			</div>
+		</div>
+	);
+};
+
+const Edit = (): JSX.Element => {
+	const blockProps = useBlockProps( {
+		className: 'wc-block-store-notices',
+	} );
+
+	return (
+		<div { ...blockProps }>
+			<StoreNotices />
```

Comment:
The `StoreNotices` component is quite simple, what do you think about putting its content here directly instead of creating a new component?

Reply:
Let's wait until we have the decision in https://github.com/woocommerce/woocommerce-blocks/pull/8157#discussion_r1071277405